biopet-tools/src/main/scala/nl/lumc/sasc/biopet/tools/GtfToRefflat.scala

```diff
@@ -124,8 +186,12 @@ object GtfToRefflat extends ToolCommand {
         if (gene.strand) "+" else "-",
         (transcript.transcriptionStart - 1).toString, //TODO: check if this is correct
         transcript.transcriptionEnd.toString,
-        transcript.codingStart.toString, //TODO: check if this is correct
-        transcript.codingEnd.toString, //TODO: check if this is correct
+        transcript.codingStart
+          .getOrElse(transcript.transcriptionEnd)
```

Comment:
Nope, if codingstart is empty it means that there is no coding region. In that case both coding positions should be set on the end of the transcript.